Makefile

```diff
@@ -0,0 +1,11 @@
+# $FreeBSD$
+
+TESTSDIR=	${LOCALBASE}/tests/tcptestsuite
+ATF_TESTS_SH=	tcptestsuite_atf_test
+TEST_METADATA=	required_programs="packetdrill"
```

Comment:
Requiring the program this way, but hardcoding the path to `packetdrill` in the generator means that if I run the test script with packetdrill in my $PATH, but not located at /usr/local/bin, the test will fail pretty quickly.

It's better to use a consistent full path to packetdrill, or not use the full path to it.

Reply:
Fixed in commit ab9393f.